index.d.ts

```diff
@@ -0,0 +1,13 @@
+import unified from 'unified'
+import unist from 'unist'
+import hast from 'hast'
```

Comment:
could these be destructured?
```suggestion
import {Transformer} from 'unified'
import {Node} from 'unist'
import {Element} from 'hast'
```

Reply:
Could these also be added to dependencies in package.json?

Reply:
I believe the consensus is to typically add TypeScript imports to dependencies, except some special cases. I.e. `@types/unist` and `@types/hast` belong in dependencies, but not `unified`, as having multiple `unified` versions in the dependency chain is usually undesirable and may cause type conflicts, and it’s typically already pulled in as a dependency when using unified plugins.